EKF/ekf.h

```diff
@@ -44,6 +44,8 @@
 
 #include "estimator_interface.h"
 
+typedef matrix::Dual<float, 4> Dual4f;
```

Comment:
I think there is a better place to put this - maybe `estimator_interface.h`

Reply:
done